src/applications/pokemon_summary_screen/main.c

```diff
@@ -13,10 +13,12 @@
 
 #include "struct_defs/struct_02099F80.h"
 
+#include "applications/pokemon_summary_screen/3d_anim.h"
 #include "applications/pokemon_summary_screen/main.h"
 #include "applications/pokemon_summary_screen/sprite.h"
 #include "applications/pokemon_summary_screen/subscreen.h"
 #include "applications/pokemon_summary_screen/window.h"
+#include "graphics/pokemon_summary_screen/pl_pst_gra.naix"
 #include "text/gmm/message_bank_pokemon_summary_screen.h"
 #include "text/pl_msg.naix"
```

Comment:
**thought**: We probably want to update the inclusion-sort rules so that things are more neatly-organized. Maybe we should start prefixing `res` in front of asset-includes?